variables.tf

```diff
@@ -333,3 +165,9 @@ variable "module_depends_on" {
   description = "(Optional) A list of external resources the module depends_on. Default is []."
   default     = []
 }
+
+variable "module_tags" {
```

Comment:
we called this `tags` in all other modules ....

Reply:
yes but `tags` are not clear enough on which resource this is applied..  can rename but ux wise i would start proposing module_tags..